src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url(../images/star.svg) center no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
```

Comment:
last child shouldn't have margin. Use `:not(:last-child)`

Reply:
https://github.com/mate-academy/layout_catalog/pull/3689/files#r1341318623
fix it please